### PR TITLE
Update README.md to reflect yaml changes

### PR DESCRIPTION
**Pull request by TomGuppy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,55 @@ These instructions will enable you to get the [Full Scan](https://github.com/zap
 ### Pre-requisites
 Docker needs to be installed on the machine the agent will be running on.
 
-### The YAML file explained
-This first stage ensures that the agent has access to the pre-defined $(Build.ArtifactStagingDirectory) location, this may or may not be needed - this is used here as we are running on Linux.
+### Incorporate into an Azure DevOps Pipeline - UKHO
+This can be incorpated into an Azure Devops Pipeline by copying the [azure-pipelines.yml](https://github.com/UKHO/owasp-zap-ui-scan/blob/master/azure-pipelines.yml) and using this within a pipeline created against your repository for running the Zap scan.
```
Comment (gregzealley) [inline README.md]: Typo - incorpOrated

File: README.md [hunk 2]
```diff
@@ -12,8 +12,55 @@ These instructions will enable you to get the [Full Scan](https://github.com/zap
...
+		      type: github
+		      endpoint: UKHO
+		      name: UKHO/owasp-zap-ui-scan
+		      ref: refs/heads/master
+	jobs:
+	- template: owasp-zap-ui-scan-template.yml@owaspzapui
+	  parameters:
+	  url: $(ApplicationUrl)
+	  
+This yaml will use the contents of the master branch for this repository, using [owasp-zap-ui-scan-template.yml](https://github.com/UKHO/owasp-zap-ui-scan/blob/master/owasp-zap-ui-scan-template.yml), [ZapTransform.ps1](https://github.com/UKHO/owasp-zap-ui-scan/blob/master/src/ZapTransform.ps1) and [ZapTransformTemplate.xslt](https://github.com/UKHO/owasp-zap-ui-scan/blob/master/src/ZapTransformTemplate.xslt) on the master branch of this repository.
```
Comment (gregzealley) [inline README.md]: You mention this twice "on the master branch of this repository. "

File: README.md [hunk 3]
```diff
@@ -12,8 +12,55 @@ These instructions will enable you to get the [Full Scan](https://github.com/zap
...
+	jobs:
+	- template: owasp-zap-ui-scan-template.yml@owaspzapui
+	  parameters:
+	  url: $(ApplicationUrl)
+	  
+This yaml will use the contents of the master branch for this repository, using [owasp-zap-ui-scan-template.yml](https://github.com/UKHO/owasp-zap-ui-scan/blob/master/owasp-zap-ui-scan-template.yml), [ZapTransform.ps1](https://github.com/UKHO/owasp-zap-ui-scan/blob/master/src/ZapTransform.ps1) and [ZapTransformTemplate.xslt](https://github.com/UKHO/owasp-zap-ui-scan/blob/master/src/ZapTransformTemplate.xslt) on the master branch of this repository. 
+
+### Incorporate into an Azure DevOps Pipeline - External to UKHO
+
+<p align="center"> <br> :construction: <b> UNDER CONSTRUCTION </b> :construction: <br><br> </p>
```
Comment (gregzealley) [inline README.md]: Maybe un-centre so its more obvs

File: README.md [hunk 4]
```diff
@@ -57,10 +104,6 @@ The chmod permissions are then reverted.
           script: 'chmod 755 -R $(Build.ArtifactStagingDirectory)'
       displayName: "Revert chmod permissions (ArtifactStagingDirectory)"
```
Comment (gregzealley) [inline README.md:106]: Should you mention about the publishing of results.. I feel like I am left hanging at the moment

Comment (TomGuppy) [reply README.md:106]: It mentions about the publishing of the results in the section above :)

File: README.md (hunk 3 above)
Comment (TomGuppy) [reply README.md]: Done 👍

File: README.md (hunk 2 above)
Comment (TomGuppy) [reply README.md]: Removed 👍

File: README.md (hunk 1 above)
Comment (TomGuppy) [reply README.md]: Corrected 👍

**Review by gregzealley — APPROVED** [review]

**Pull request merged**